scripts/functions/terraform-functions.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
```

Comment:
Maybe rebase because I think this has been replaced / removed. The scripts to be used are the ones under scripts/workflows as these are the ones that will get checked out as part of the sparse checkout to the domain repo calling all these workflows in uec-dos-management. But I think to avoid scripts calling scripts these functions have been moved / refactored